newsfragments/143.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix double-acquire by in the executor that could lead to deadlock
```

Comment:
This will show up in the release notes; please try a little harder to be detailed and clear. One possibility is shown.

```suggestion
`TrioExecutor` now acquires just one token from its `~trio.CapacityLimiter` for each
worker thread that it runs, rather than two. The double-acquisition previously could
cause a deadlock when trying to run a large number of tasks simultaneously using
:meth:`asyncio.loop.run_in_executor`.
```